Classes/Configuration/ExtensionBuilderConfigurationManager.php

```diff
@@ -198,9 +195,9 @@ public static function getExtensionBuilderJson($extensionKey, $storagePath = nul
         $jsonFile = $storagePath . $extensionKey . '/' . self::EXTENSION_BUILDER_SETTINGS_FILE;
```

Comment:
There is still a reference to PATH_typo3conf 
in line 194.
Can you change it:
$storagePath = $storagePath ?? PATH_typo3conf . 'ext/';
$storagePath = $storagePath ?? Environment::getPublicPath()  . '/typo3conf/ext/';

Reply:
Can also be done after merging